scripts/Dockerfile

```diff
@@ -10,29 +10,36 @@ RUN mkdir -p /bins
 RUN cp target/wallet-proxy /bins/wallet-proxy
 
 # Fetch identity-providers-with-metadata.json and global.json from genesis data.
-WORKDIR /
 RUN mkdir -p -m 0600 ~/.ssh && ssh-keyscan gitlab.com >> ~/.ssh/known_hosts
 # Clone specified ref of genesis-data (master by default).
 # The only files that are needed are `identity-providers-with-metadata.json` and `global.json`.
 # These change rarely and are always manually generated and curated.
 # It is important that those files match the identity providers and global parameters of the network
 # the wallet-proxy will connect to. This needs to be ensured manually.
-ARG GENESIS_REF=master
-RUN --mount=type=ssh git clone --depth 1 --branch ${GENESIS_REF} git@gitlab.com:Concordium/genesis-data.git
+ARG genesis_ref
+ARG genesis_path
+WORKDIR /tmp
+RUN --mount=type=ssh git clone --depth 1 --branch "${genesis_ref}" git@gitlab.com:Concordium/genesis-data.git
+COPY "genesis-data/${genesis_path}/generated-data" /genesis-data
```

Comment:
Did we not agree that genesis data repository should not be used in building the image.

All of the files that are needed should be part of deployment.

Reply:
I don't recall agreeing on that, but it's fine by me. Removing all genesis stuff.